src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -1157,18 +1157,8 @@ internal static DeclarationModifiers GetModifier(SyntaxKind kind, SyntaxKind con
                     return DeclarationModifiers.None;
             }
         }
-
-        private bool IsPossibleModifier()
-        {
-            return IsPossibleModifier(this.CurrentToken);
-        }
-
-        private static bool IsPossibleModifier(SyntaxToken token)
```

Comment:
@OmarTawfik Was there a particular reason to remove this helper? If not, may I add it in my PR and update usages?

Reply:
@alrz not sure which PR you're working on, but if it improves code (you need the same logic in other places) and you can update usages, I don't see a reason why not :)

Reply:
thanks. (updated the PR at https://github.com/dotnet/roslyn/pull/23533)